toolbox

```diff
@@ -165,6 +165,10 @@ list() {
 }
 
 container_create() {
+    VOLUMES=""
```

Comment:
While we do this, I think we could put all the `--volume`-s in a variable. But I can do it myself in another patch